github-dark.css

```diff
@@ -1989,7 +1989,7 @@
   }
   .text-pending, a.text-pending, .compare-pr-placeholder p,
   .compare-pr-placeholder .help-link, .text-pending strong {
-    color: #cc5 !important;
+    color: #ddd !important;
   }
```

Comment:
So is this OK? or shall I use some the brown color for border... I dont even know what that is in this 3 digit hex format...